tools/repl.md

```diff
@@ -53,6 +53,51 @@ Uncaught AssertionError
     at <anonymous>:2:1
 ```
 
+## `--eval-file` flag
+
+`--eval-file` flag allows you to run code from specified files before you are
+dropped into the REPL. Like the `--eval` flag, this is useful for importing code
+you commonly use in the REPL, or modifying the runtime in some way.
+
+Files can be specified as paths or URLs. URL files are cached and can be
+reloaded via the `--reload` flag.
+
+If `--eval` is also specified, then `--eval-file` files are run before the
+`--eval` code.
+
+```
+$ deno repl --eval-file=https://examples.deno.land/hello-world.ts,https://deno.land/std@0.135.0/encoding/ascii85.ts
```

Comment:
Replace hardcoded version of `deno_std` with `@$STD_VERSION` makers, they will be automatically filled with the latest available version